dataAccessObjects.py

```diff
@@ -24,9 +24,12 @@ def queryOneValue(self,sql):
 
 class DataManipulateDao(DefaultDao):
     #TODO handle if derived class not assign dataName
-    def markExpired(self,targetId):
+    def markExpired(self,targetId,markOldData):
```

Comment:
The argument `markOldData` should be `markOldData=None`.
Since the original code used this function would lack of one argument "markOldData".